guide/case-studies/blixt-wallet.md

```diff
@@ -0,0 +1,154 @@
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-header-mobile@2x.jpg"
+   alt-text = "Smartphones with various Blixt wallet UI screens"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Blixt wallet
+
+## How do we help users new to lightning start using their wallet?
+
+The [Bitcoin Design Community](https://bitcoindesign.slack.com/join/shared_invite/zt-10sxfovaq-isViijl4RThKRs_TsAQnuA#/shared-invite/email) ran a design sprint on an open-source lightning wallet and during this sprint we collaboratively focused on re-designing one aspect of the user flow, the **onboarding journey**. During this collaborative effort, [we got together](https://www.youtube.com/watch?v=nctibe_a4rM&list=PLpV0KfVOMojYywq1Blp3SGx62cSyoRfHt&index=3) and as a group of designers and researchers we aimed to improve the user experience by using good design principles incorporating as much feedback from users as possible. 
+
+<p class="h3">Blixt Wallet is a non-custodial open-source Bitcoin Lightning Wallet for Android and iOS and aims to serve users who are new to lightning, but are familiar with bitcoin and want to try out the lightning network for the first time.</p>
+
+The current user base of Blixt tends to be more advanced users and Blixt would like to move towards catering more to users who want to try out the lightning network as well. 
+
+What does Blixt do?
+
+- Onboard someone quickly over lightning so they can have a channel already provisioned to receive funds
+- Automatically open channels anytime the user has on-chain funds (Auto pilot)
+- Automatic channel creation (using [Dunder LSP](https://github.com/hsjoberg/dunder-lsp)) during initial deposit of funds
+- Support for custom LSP
+
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-users.jpg"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-users@2x.jpg"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-users-mobile.jpg"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-users.jpg"
+   alt-text = "Characters representing Blixt's users"
+   caption = "Illustrations: [OpenPeeps](https://www.openpeeps.com/) by Pablo Stanley"
+   width = 800
+   height = 450
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process.jpg"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process@2x.jpg"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process-mobile.jpg"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process.jpg"
+   alt-text = "Diagram of stages in the Blixt UX process"
+   width = 800
+   height = 929
+%}
+
+Since Blixt is an open source wallet we collected user research simply by going on their Discord channel and collecting comments and questions from users.
```

Comment:
Simplifying the sentence 
```suggestion
Since Blixt is an open source wallet, we collected user research simply by going on their Discord channel and collecting user comments and questions.
```